### ISSUE TITLE
Octave issue

### ISSUE THREAD

**Issue by yaras6** [CoderLine/alphaTab#141]
# Your environment

* Version used: (master branch)
* Platform used: JavaScript
* Rendering engine used: SVG (default)
* Browser Name and Version: 58.0.3029.110 (64-bit)
* Operating System and version Window 10 desktop:
* Link to your project: http://tuneguitar.net/en

# Expected Results

Render track in same as in previous versions.

# Observed Results

Some tracks are rendered one octave higher. Instead 2nd thread is 14th, insted 4th is 16th etc. 

# Steps to Reproduce (for bugs)

Please download song and load it with newest alphaTab version 
1. http://tuneguitar.net/pl/Player/DownloadSong?songId=18480&type=1

Proper result of rendering with old version is available on
1. http://tuneguitar.net/pl/Player?songId=18480&type=1

Currently same song looks like this
![image](https://user-images.githubusercontent.com/25822217/27002306-eea5eb06-4dde-11e7-8b26-d929cfcc653a.png)

Thanks in advance for your help. Did you changed any core functionality?

**Comment by yaras6** [comment]
On your fiddle issue exists too. Please increase count to see the issue. Probably issue exists in all or most tabs
https://jsfiddle.net/danielku15/jo64pgLp/

**Comment by Danielku15** [comment]
Thanks for the report. 

Looks like a side effect of #139 In general it is correct that the standard music notation is 1 octave lower on Guitar instruments. I corrected the behavior that for non-guitar instruments the standard notation is 1 octave higher. This behavior is then consistent with Guitar Pro. 

As part of the last rework in #139 I added 2 separate transposition pitches. The first one is used for transposing the standard notation, the second one does a full transposition of the music notation (including tabs and audio playback). At 1-2 positions I used the wrong transposition value which caused the tabs to have the wrong values. This is fixed now. 

Sadly the visual rendering is really hard to unit test. I need to find a good solution for this in future to prevent such bugs.
